src/OctoshiftCLI.IntegrationTests/TestHelper.cs

```diff
@@ -86,6 +86,9 @@ public async Task ResetGithubTestEnvironment(string githubOrg)
 
             foreach (var repo in githubRepos)
             {
+                _output.WriteLine($"Deleting migration log for repo: {githubOrg}\\{repo}");
+                DeleteMigrationLog(githubOrg, repo);
```

Comment:
Even though we only need to delete the logs for the target GitHub org, I thought for our INT tests it'd be just easier to shove this here, it won't do anything for the source GitHub org since it wouldn't find any logs.